File: exam/src/main/java/in/testpress/exam/ui/view/SlidingPaneLayout.java
package in.testpress.exam.ui.view;

import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;

public class SlidingPaneLayout extends android.support.v4.widget.SlidingPaneLayout {

    public SlidingPaneLayout(Context context) {
        super(context);
    }

    public SlidingPaneLayout(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public SlidingPaneLayout(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    @Override
    public boolean onInterceptTouchEvent(MotionEvent ev) {
        return false;
    }

    @Override
    public boolean onTouchEvent(MotionEvent event) {
        return false;
    }
}

File: core/src/main/java/in/testpress/ui/view/SlidingPaneLayout.kt
package `in`.testpress.ui.view

import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import androidx.slidingpanelayout.widget.SlidingPaneLayout

class SlidingPaneLayout : SlidingPaneLayout {
    constructor(context: Context?) : super(context!!) {}
    constructor(context: Context?, attrs: AttributeSet?) : super(context!!, attrs) {}
    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(context!!, attrs, defStyle) {}

    override fun onInterceptTouchEvent(ev: MotionEvent): Boolean {
        return false
    }

    override fun onTouchEvent(event: MotionEvent): Boolean {
        return false
    }
}
